R/candid.r

```diff
@@ -0,0 +1,58 @@
+valid_blood_groups = c('O', 'A', 'B', 'AB')
+
+# Tests if element is a valid Blood Group character
+blood_group_checker <- function(input_string){
+  if (input_string %in% valid_blood_groups){
+    return(TRUE)
+  }
+  return(FALSE)
+}
```

Comment:
Once this is all done, we should identify which functions repeat and move them all to a utility file.